code/weapon/beam.h

```diff
@@ -90,6 +99,7 @@ typedef struct fighter_beam_fire_info {
 	int warmdown_stamp;
 	float life_left;	
 	float life_total;
+	int  fire_method;
```

Comment:
This is never used, correct? I don't see it set or checked against anywhere.

Actually, I'm not sure why this struct even still exists.

Reply:
It's used for `beam_fire_targeting()` which is a weird very niche case for beams. It handles safety well enough on its own so yeah, this isn't really necessary. It could probably be refactored as a special case of regular `beam_fire`, but one thing at a time.